stores/json/store.go

```diff
@@ -42,15 +42,30 @@ func (store BinaryStore) EmitEncryptedFile(in sops.Tree) ([]byte, error) {
 	return store.store.EmitEncryptedFile(in)
 }
 
+type BinaryStoreEmitPlainError struct {
+	message string
+}
+
+func (e BinaryStoreEmitPlainError) Error() string {
+	return e.message
+}
+
 // EmitPlainFile produces plaintext json file's bytes from its corresponding sops.TreeBranches object
 func (store BinaryStore) EmitPlainFile(in sops.TreeBranches) ([]byte, error) {
+	if len(in) != 1 {
+		return nil, BinaryStoreEmitPlainError{message: "There must be exactly one tree branch"}
```

Comment:
Just for future reference for other Go projects, and not applicable here:

1. Normally in Go, error strings should not be capitalized (https://github.com/golang/go/wiki/CodeReviewComments#error-strings). However, that ship has sailed in this code base :-).
2. The more idiomatic name would be something along the lines of `ErrEmitPlain`. However, I can see you probably based this on `NotImplementedError`, `decryptGroupError`, etc. already existing in the code base.